external_push/templates/external_push/brewfather_push_target_add.html

```diff
@@ -45,6 +53,26 @@ <h3 class="form-header">Push Target Settings</h3>
 {% block scripts %}
 <script>
 
+function DevicePushSelectionHide() {
+console.log(document.getElementById('id_device_type').options[document.getElementById('id_device_type').selectedIndex].value) 
+    if(document.getElementById('id_device_type').options[document.getElementById('id_device_type').selectedIndex].value == "gravity") {
+console.log("A")
```

Comment:
Are the console.log statements left over from testing? I'm fine with them, TBQH, but if you want to leave them, it would be good to update the messages to be a bit more descriptive.

Reply:
I missed to remove them, I just put them in to see if the code was run correctly. Please remove the log statements.